libqfieldsync/utils/qgis.py

```diff
@@ -48,7 +48,9 @@ def open_project(filename: str, filename_to_read: Optional[str] = None) -> bool:
     return is_success
 
 
-def make_temp_qgis_file(project: QgsProject) -> str:
+def make_temp_qgis_file(
+    project: QgsProject,
+) -> str:
```

Comment:
```suggestion
def make_temp_qgis_file(project: QgsProject) -> str:
```
No need for this diff at all.